examples/simple/src/authProvider.tsx

```diff
@@ -60,10 +62,16 @@ export default {
         return Promise.resolve(role ?? '');
     },
     getIdentity: () => {
+        const id = localStorage.getItem('login');
```

Comment:
No, we want the user to be authenticated by default in the simple example (see `checkAuth`), this will break it. 
Return the default identity if there is no login in localStorage.

Reply:
As users are authenticated by default, this is just to make TS happy but ok